### PR TITLE
fix #75599 imginn.com

### LINKED ISSUE #75599: imginn.com
### Issue URL (Annoyance)
 [https://imginn.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fimginn.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v        1.8.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media, <br/>Fanboy's Social Blocking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#75599

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -550,6 +550,9 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75599
+imginn.com#%#//scriptlet("abort-current-inline-script", "document.createElement", "unblocker")
+imginn.com#@#.ad_slot
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:555]: `||copyrightcontent.org/unblocker/$domain=imginn.com` is ok. Also we have a similar rule, so just add a domain.
As I said earlier. don't add rules at the beginning, just a little lower.
For example
<details><summary>Screenshot:</summary>

image

</details><br/>

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:555]: @zloyden What is wrong If I use scriptlet?

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:555]: Everything works perfectly even for content blockers.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: It's not necessary here. The best option is considered to be a simpler solution.

**Pull request merged**